Libraries/src/Amazon.Lambda.Annotations/Amazon.Lambda.Annotations.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <AssemblyVersion>0.10.0.0</AssemblyVersion>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
```

Comment:
Because the new `HttpResults` class uses `System.Text.Json` for JSON serialization that is not available for .NET Standard 2.0 without taking a dependency. Since `System.Text.Json` is only used at runtime, not source generator time, we can avoid the dependency by just using the newer APIs for .NET 6 during runtime.

Reply:
Got it, thanks.